sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -135,6 +135,15 @@ export interface CertificateOperationError {
     readonly message?: string;
 }
 
+// @public
+export type CertificateOperationPollerLike = PollerLike<CertificateOperationPollPublicState, KeyVaultCertificateWithPolicy>;
+
+// @public
+export interface CertificateOperationPollPublicState extends PollOperationState<KeyVaultCertificateWithPolicy> {
```

Comment:
I'm not sure if we want this? `PollOperationState` is already retrievable via `getOperationState()`, so duplicating it in `getState()` seems unnecessary. 

I'd rather keep the "state" exposed by PollerLike distinct from PollOperationState. This has the benefit of meaning you can have the "private" poll state have a public property you slice off instead of having to manufacture a separate object for the public state.

Also I think naming wise we didn't want to call something "public" in the public interface

Reply:
I'll resolve this for now.